src/iterative_sorting/iterative_sorting.py

```diff
@@ -1,30 +1,58 @@
 # TO-DO: Complete the selection_sort() function below 
 def selection_sort( arr ):
     # loop through n-1 elements
+    if len(arr) < 2:
+        return arr
     for i in range(0, len(arr) - 1):
         cur_index = i
         smallest_index = cur_index
         # TO-DO: find next smallest element
         # (hint, can do in 3 loc) 
+        for j in range(cur_index, len(arr)):
+            if arr[j] < arr[smallest_index]:
+                smallest_index = j
              
 
 
 
         # TO-DO: swap
-
-
+        arr[smallest_index], arr[cur_index] = arr[cur_index], arr[smallest_index]
 
 
     return arr
 
-
 # TO-DO:  implement the Bubble Sort function below
 def bubble_sort( arr ):
-
+    if len(arr) < 2:
+        return arr
+    d = True
+    while d:
+        for i in range(1, len(arr) ):
+            if i == 1:
+                d = False
+            if arr[i] < arr[i - 1]:
+                arr[i], arr[i - 1] = arr[i - 1], arr[i]
+                d = True
     return arr
 
 
 # STRETCH: implement the Count Sort function below
 def count_sort( arr, maximum=-1 ):
-
-    return arr
\ No newline at end of file
+    if maximum < 0:
+        maximum = 65535
```

Comment:
`max(arr)` will also work here and is likely better suited for different array values. Great job